### PR TITLE
Allow to create new variants by the grid only in full screen mode

### PR DESCRIPTION
**Pull request by poulch**
## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [ ] 🛠 Workflow CI/CD changes

## Related Issues or Documents

- closes #

## Have you written tests?
- [ ] Yes!
- [ ] No... here is why: _Writing tests are mandatory, please replace this text with why test are not included in this PR_

### CODE REVIEW

File: src/components/Datagrid/Datagrid.tsx [hunk 1]
```diff
@@ -93,19 +95,17 @@ export interface DatagridProps {
   actionButtonPosition?: "left" | "right";
   recentlyAddedColumn?: string | null; // Enables scroll to recently added column
   onClearRecentlyAddedColumn?: () => void;
+  renderHeader?: (props: DatagridRenderHeaderProps) => ReactNode;
```
Comment (Cloud11PL) [inline src/components/Datagrid/Datagrid.tsx:98]: suggestion: It seems that `renderHeader` is always present so I think we can remove `?`

**Review by Cloud11PL — DISMISSED** [review]

File: src/components/Datagrid/Datagrid.tsx (hunk 1 above)
Comment (poulch) [reply src/components/Datagrid/Datagrid.tsx:98]: No it's not, because renderHeader exist only in datagrid with variants in product details page

**Review by Cloud11PL — DISMISSED** [review]

File: .changeset/orange-bulldogs-smoke.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+Datagrid on product variants page when click add variant redirect to create new variant page instead of create new row in datagrid. Creating new row in datagrid is only possible in fullscreen mode
```
Comment (lkostrowski) [inline .changeset/orange-bulldogs-smoke.md:5]: Please improve this chagelog and proof read grammar

Comment (andrzejewsky) [inline .changeset/orange-bulldogs-smoke.md:5]: _Adding the new variant as a row is no longer possible from the product details. This means add new variant button now leads you to the variant creation page_

**Review by andrzejewsky — DISMISSED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Review by yellowee — APPROVED** [review]

**Pull request merged**